deploy-manage/upgrade/ingest-components.md

```diff
@@ -12,6 +12,12 @@ applies_to:
 Once you've successfully [upgraded your deployment or cluster](/deploy-manage/upgrade/deployment-or-cluster.md), the final step is to update your ingest components and clients in the following order: 
```

Comment:
I believe this page needs a `serverless` label - we should validate with the ingest team.